plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/deformable_convolution.cpp

```diff
@@ -0,0 +1,316 @@
+    ::testing::ValuesIn(deformableGroup),                  //
+    ::testing::Values(ngraph::op::PadType::SAME_UPPER)     //
+);
+
+const auto conv3DParams_SameLower = ::testing::Combine(    //
+    ::testing::ValuesIn(kernels3d),                        //
+    ::testing::Values(std::vector<size_t>({2, 2, 2})),     //
+    ::testing::Values(std::vector<ptrdiff_t>({0, 0, 0})),  //
+    ::testing::Values(std::vector<ptrdiff_t>({0, 0, 0})),  //
+    ::testing::ValuesIn(dilations3d),                      //
+    ::testing::Values(2),                                  //
+    ::testing::ValuesIn(group),                            //
+    ::testing::ValuesIn(deformableGroup),                  //
+    ::testing::Values(ngraph::op::PadType::SAME_LOWER)     //
+);
+
+INSTANTIATE_TEST_CASE_P(Convolution3D_ExplicitPadding, DeformableConvolutionLayerTest,
+                        ::testing::Combine(                                            //
+                            conv3DParams_ExplicitPadding,                              //
+                            ::testing::ValuesIn(netPrecisions),                        //
+                            ::testing::Values(std::vector<size_t>({1, 1, 5, 5, 5})),   //
+                            ::testing::Values(std::vector<size_t>({1, 24, 6, 6, 6})),  //
+                            ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)),       //
+                        DeformableConvolutionLayerTest::getTestCaseName);
+
+INSTANTIATE_TEST_CASE_P(Convolution3D_AutoPadValid, DeformableConvolutionLayerTest,
+                        ::testing::Combine(                                            //
+                            conv3DParams_AutoPadValid,                                 //
+                            ::testing::ValuesIn(netPrecisions),                        //
+                            ::testing::Values(std::vector<size_t>({1, 1, 5, 5, 5})),   //
+                            ::testing::Values(std::vector<size_t>({1, 24, 4, 4, 4})),  //
+                            ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)),       //
+                        DeformableConvolutionLayerTest::getTestCaseName);
+
+INSTANTIATE_TEST_CASE_P(Convolution3D_SameUpper, DeformableConvolutionLayerTest,
+                        ::testing::Combine(                                            //
+                            conv3DParams_SameUpper,                                    //
+                            ::testing::ValuesIn(netPrecisions),                        //
+                            ::testing::Values(std::vector<size_t>({1, 1, 4, 4, 4})),   //
+                            ::testing::Values(std::vector<size_t>({1, 24, 4, 4, 4})),  //
+                            ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)),       //
+                        DeformableConvolutionLayerTest::getTestCaseName);
+
+INSTANTIATE_TEST_CASE_P(Convolution3D_SameLower, DeformableConvolutionLayerTest,
```

Comment:
Can you prefix at least one of the test names with `smoke_` so at least one DeformableConvolution gets run during `check-smoke`? I appreciate that you're not cluttering `check-smoke` with lots of tests, but every op that we can test at all should have at least one test in `check-smoke`.

Reply:
I will add the test names with `smoke_`.

Reply:
Done